src/game_battler.cpp

```diff
@@ -803,3 +863,21 @@ void Game_Battler::ShiftAttributeRate(int attribute_id, int shift) {
 		--old_shift;
 	}
 }
+
+float Game_Battler::GetAttributeMultiplier(const std::vector<bool>& attributes_set) const {
+	float physical = -1001, magical = -1001;
```

Comment:
This is confusing because we have then GetAttributeMultiplier in BattleAlgorithm (commit "3.16: RM2000: Attribute modifier is the maximum of the percentage da" of #1448 and in Game_Battler... :/   (and with the same -1001 issue)

Reply:
see https://github.com/EasyRPG/Player/pull/1448/commits/8ef8f2e9d95b391373e211b49f57d6a11a74d229